### ISSUE TITLE
Replicating to embedded libSQL stopped working

### ISSUE THREAD

**Issue by penberg** [libsql/sqld#617]
When attempting to sync an embedded libSQL database, nothing happens: no data or errors. This seems to be a `sqld` bug because commit `3cbba1c8c15191db3ba9c5268aa37429033ceab2` works fine.

**Comment by haaawk** [comment]
`3cbba1c8c15191db3ba9c5268aa37429033ceab2` works
`87492b3a406c94cecd9df5bb6db9294eeb058934` does not

**Comment by LucioFranco** [comment]
This is due to us not sending the namespace message (and probably need to sync the proto files this should be fixed once we merge the repos). The reason sqld doesn't error out is that if protobuf doesn't send the tagged field it will use the default which in this case is an empty byte array which doesn't cause an immediate failure. So there are two fixes here to check for that and to actually send the namespace.

**Comment by MarinPostma** [comment]
@LucioFranco is this solved?

**Comment by LucioFranco** [comment]
Yeah this was fixed in #619 where if there is no metadata it just uses the default namespace. (no need for clients to upgrade their proto version)
